Tests/RegistryTest.php

```diff
@@ -24,54 +25,57 @@
 
 class RegistryTest extends TestCase
 {
+    /** @var MockObject&Container  */
+    private MockObject $container;
+
+    protected function setUp(): void
+    {
+        $this->container = $this->createMock(Container::class);
```

Comment:
If you want to reduce the diff, you could simply replace `$container = $this->getMockBuilder(ContainerInterface::class)->getMock();` with `$container = $this->createMock(Container::class);` without also moving it to an instance property (which does not have much benefit as there is no shared setup logic

Reply:
Thanks but I know. My goal wasn't to reduce the diff.